components/RegisterFormChild.js

```diff
@@ -46,7 +46,6 @@ export default function RegisterFormChild({
     let {name, value} = event.target;
```

Comment:
Better use `const` instead of `let` at this point:

```suggestion
    const {name, value} = event.target;
```